README.md

```diff
@@ -153,9 +153,90 @@ This library doesn't handle this for you, but here are the 2 main options:
 This is the easiest method: by rendering your form inside a live component,
 it will automatically re-render when the form changes.
 
-### B) Write custom JavaScript
+### B) Use [Symfony UX Turbo](https://symfony.com/bundles/ux-turbo/current/index.html#decomposing-complex-pages-with-turbo-frames)
+
+By using turbo frames you can have a dynamic form running quickly without any JavaScript.
```

Comment:
```suggestion
By using Turbo Frames you can have a dynamic form running quickly without any JavaScript.
```

Reply:
Hm, should we also mention that you have to use Turbo on your website? Right now it sounds like I just need to write the code below and it should magically work, but if people do not have Turbo activated on that page - that will not work, right?

Reply:
Yes indeed, and more than that, we need to explain that only Turbo Frame is needed, not Turbo Drive (many of my colleague do not know the difference). I'll rephrase it to make it clear.